components/dialog/test/dialog-ifrau-contents.vdiff.html

```diff
@@ -18,24 +18,13 @@
 <body class="d2l-typography">
 	<d2l-button id="open">Open Dialog</d2l-button>
```

Comment:
I left this `d2l-button` here since this test requires a whole `html` file anyways, but I can remove it if we want to be consistent with the look of the other tests.  Left in, the `dialog-ifrau` tests result in no diffs 🎉